### ISSUE TITLE
Bridge package docs: `flame_svg`

### ISSUE THREAD

**Issue by spydon** [flame-engine/flame#1964]
# Bridge package docs for `flame_svg`

This bridge package should have its own documentation section on the docs site, similarly to `flame_audio` for example:
https://docs.flame-engine.org/main/flame_audio/flame_audio.html

For inspiration (and copy-pasting) you can have a look at the documentation for the package written in the `README.md` file and the documentation that the package is bridging too.

# Good to know

Make sure that you comment on the issue that you will be working on this before starting to program.

To set up the Flame environment you need to follow the instructions in our [contributing.md](https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md#environment-setup) doc, if you have any problems setting up your environment (or other issues of course) feel free to ask for help in the Flame channel on our [Discord server](https://discord.com/invite/pxrBmy4).

Since this is a documentation change, you need to be able to build the docs. You can do that by running `melos doc-build` after you have followed the instructions in the contributing.md doc and also [installed Sphinx on your system](https://www.sphinx-doc.org/en/master/usage/installation.html). After you have built the docs you can open the `doc/_build/html/index.html` file in your web browser to see the changes.

Make sure to include a video of the example when you upload the PR so that it will be easier for us to review it. (You can attach videos directly in the PR, so no need to upload them somewhere else)

**Happy hacking!**

**Comment by trunghq3101** [comment]
Working on this!

**Comment by spydon** [comment]
> Working on this!

Super! I've assigned you to the issue, remember to join our Discord server if you have any questions etc. :)

**Comment by TheZal** [comment]
Is this still available? I can work on that!

**Comment by trunghq3101** [comment]
@TheZal Yes, currently I'm working on another issue. So, freely take it from me.

**Comment by TheZal** [comment]
Perfect!

**Comment by spydon** [comment]
> I'm getting an error: `Could not find a command named "sphinxdoc".` I tried to run the command inside the flame directory and inside the flame/doc directory. I followed the [contributing guide](https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md#environment-setup) and got no errors

Read the "Good to know" section in the issue description.

**Comment by TheZal** [comment]
I did everything in the guide and also installed `sphinx` without any problem. The `melos bootstrap` ran fine but after that  I just can't run the `melos sphinxdoc` command

**Comment by ufrshubham** [comment]
> I did everything in the guide and also installed `sphinx` without any problem. The `melos bootstrap` ran fine but after that  I just can't run the `melos sphinxdoc` command

The command is `melos doc-build`. Not sure from where you got `melos sphinxdoc`

**Comment by ufrshubham** [comment]
Oh, my bad. Just saw the `Good to know ` section. I think it is either wrong or is referring to some old command.

**Comment by TheZal** [comment]
Yeah, now the command ran but got this error: `Extension error: Could not import extension myst_parser (exception: No module named 'myst_parser')`.
Btw inside the doc directory i can't find `doc/_build/html/index.html`. There's not even the _build folder.
I got this inside the doc folder
![image](https://user-images.githubusercontent.com/12339188/194874378-072b2d15-c898-42f8-891e-25205a0a7974.png)

**Comment by ufrshubham** [comment]
> Yeah, now the command ran but got this error: `Extension error: Could not import extension myst_parser (exception: No module named 'myst_parser')`.


Did you install all the python packages listed in the `requirements.txt` located at _sphinx?

> Btw inside the doc directory i can't find `doc/_build/html/index.html`. There's not even the _build folder.
> I got this inside the doc folder
> ![image](https://user-images.githubusercontent.com/12339188/194874378-072b2d15-c898-42f8-891e-25205a0a7974.png)
> 

`_build` will be created while the actual doc build starts. In your case it is failing because a required package is not found.

**Comment by TheZal** [comment]
Now I got it, thx, I'm new at this!

**Comment by TheZal** [comment]
Hi @spydon, this was tagged as hacktoberfest-special when I saw it the first time, is this still eligible for the hacktoberfest special pack?

**Comment by spydon** [comment]
> Hi @spydon, this was tagged as hacktoberfest-special when I saw it the first time, is this still eligible for the hacktoberfest special pack? 

It is not unfortunately, I had misunderstood what the `hacktoberfest-special` tag did so I accidentally put it on all of our issues.

**Comment by TheZal** [comment]
Yeah I saw it now, no problem, my bad! BTW this is still an hacktoberfest-accepted eligible pr right? So where I can claim the swag? 
![image](https://user-images.githubusercontent.com/12339188/200785397-56eae46d-acc3-4727-a2e9-4084d3f3e0e4.png)
